.devcontainer/README.md

```diff
@@ -2,6 +2,40 @@
 
 For a comprehensive developer environnment supporting AGW c / c++ workflows (extensible with ~effort to other Magma uses) - see the instructions below.
 
+## Prerequisites and Restrictions
+
+DevContainer is a technology developed by Microsoft. Thus, it is only available for [_VS Code_](https://code.visualstudio.com/docs/remote/containers), but **not** for _IntelliJ_. However, as Github is also currently owned by Microsoft, Github provides a DevContainer integration named ["Codespaces"](https://docs.github.com/en/codespaces).
+In order to use the Github codespaces, you must be member of the Magma Github organisation.
```

Comment:
For Codespaces, we have a doc here so we should just link to it. https://docs.magmacore.org/docs/next/contributing/contribute_vscode